ios/browser/api/ipfs/ipfs_api.h

```diff
@@ -15,8 +15,8 @@ OBJC_EXPORT
 @protocol IpfsAPI
 @required
 
-@property(nonatomic, nullable) NSURL* nftIpfsGateway;
-@property(nonatomic, nullable) NSURL* ipfsGateway;
+@property(readonly, nullable) NSURL* nftIpfsGateway;
+@property(readonly, nullable) NSURL* ipfsGateway;
 
```

Comment:
we don't need those two properties anymore since we are not reading and displaying anywhere in the wallet settings.